src/detail-page/DetailPage.tsx

```diff
@@ -20,14 +21,15 @@ export const TabContainer = styled('div')`
 `;
 
 export const DetailPage: React.FC<{}> = () => {
-  const { professionId } = useParams<{professionId: string}>();
+  const { professionId } = useParams<{ professionId: string }>();
   const profession = ITProfessions.find(p => p.id === professionId);
```

Comment:
I think it would be good to redirect to the error page, when there is no profession.

Reply:
done